cloud-etl/config/demo.cfg

```diff
@@ -0,0 +1,37 @@
+# Kinesis
+
+# KINESIS_STREAM_NAME: will be created by the demo
+export KINESIS_STREAM_NAME='demo-logs'
+
+export KINESIS_REGION='us-west-2'
+
+# AWS_PROFILE: profile must exist in ~/.aws/credentials
+export AWS_PROFILE=default
+
+
+
+# Cloud storage sink
+
+# DESTINATION_STORAGE can be one of 's3' or 'gcs' or 'az'
+export DESTINATION_STORAGE='az'
+
+export STORAGE_REGION='us-west-2'
```

Comment:
Usually I see Azure regions formatted as uswest2 - interesting that this format also works!